tests/modules/asset_groups/resources/utils.py

```diff
@@ -299,5 +303,5 @@ def clone_asset_group(
     clone = CloneAssetGroup(client, admin_user, owner, guid, force_clone)
 
     if not expect_failure:
-        assert clone.response.status_code == 200
+        assert clone.response.status_code == 200, clone.response.data
```

Comment:
I think I may have removed this as I thought it was an error. What's the advantage of passing clone.response.data to the AssertionError ?

Reply:
Ah, it's to show the response when assertion error fails:

```python
>>> assert 201 == 200
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError
```

vs

```python
>>> assert 201 == 200, 'a message'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError: a message
```